substrate/bin/node/executor/Cargo.toml

```diff
@@ -1,13 +1,12 @@
 [package]
-name = "node-executor"
+name = "polkadot-node-executor"
```

Comment:
Don't do this. This crate should not be released!

Reply:
Why not release it? `try-runtime-cli` currently uses the `ExecutorDispatch` struct. 

Maybe we should just move it back into the monorepo (as a seperate binary) like you suggested earlier..

Reply:
Maybe we should move it into the monorepo as you suggested to avoid these headaches.. But it still seems that it is reasonable some projects may want to use these exported items.

Reply:
`sc-executor` != `node-executor`. Node executor being the executor declaration for the Substrate node that is nothing that should be released.

Reply:
`sc-executor` is already released.

Reply:
Ah.. So it turns out `ExecutorDispatch` was being imported from `node-executor`. `ExecutorDispatch` is also available from `sc-executor`, so we can remove `node-executor` as a dependency. I then got `sc-executor` and `node-executor` confused, leading to my confused msg above.

Thank you for flagging this @bkchr. I'll revert my previous commit here and remove `node-executor` as a dependency from `try-runtime-cli.